lib/src/io/node.dart

```diff
@@ -76,15 +94,15 @@ Future<String> readStdin() async {
   });
   // Node defaults all buffers to 'utf8'.
   var sink = utf8.decoder.startChunkedConversion(innerSink);
-  process.stdin.on('data', allowInterop(([Object? chunk]) {
+  process?.stdin.on('data', allowInterop(([Object? chunk]) {
```

Comment:
You can avoid nullability issues here by replacing the `isNode` check at the beginning with

```dart
var process_ = process;
if (process_ == null) {
  throw UnsupportedError("readStdin() is only supported on Node.js");
}
```

and then just referring to `process_` here, which the Dart compiler should understand as non-nullable.

Reply:
Fixed in https://github.com/oddbird/dart-sass/commit/1937affd6fa9fc3965c72222f323f56fb53352d7